solutions/src/test/java/ru/sberbank/school/task13/cacheproxy/testclasses/NonSerializebleObject.java

```diff
@@ -0,0 +1,27 @@
+package ru.sberbank.school.task13.cacheproxy.testclasses;
+
+import java.util.Objects;
+
+public class NonSerializebleObject {
+    String a;
+    String b;
+
+    public NonSerializebleObject(String a, String b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof NonSerializebleObject)) return false;
+        NonSerializebleObject that = (NonSerializebleObject) o;
+        return Objects.equals(a, that.a) &&
```

Comment:
&#39;&amp;&amp;&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1617704471>*